src/Lib/ColorPicker.vala

```diff
@@ -0,0 +1,332 @@
+/*
+* Copyright (c) 2019-2020 Alecaddd (https://alecaddd.com)
+*
+* This file is part of Akira.
+*
+* Akira is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+
+* Akira is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+
+* You should have received a copy of the GNU General Public License
+* along with Akira. If not, see <https://www.gnu.org/licenses/>.
+* 
+* Authored by: Ivan "isneezy" Vilanculo <vilanculoivan@gmail.com>
+* Ported from: https://github.com/ColorPicker/RonnyDo
+*/
+
+public class Akira.Lib.ColorPicker : Gtk.Window {
```

Comment:
This feels more like a utility tool which we might start using in multiple locations.
Let's move this into `/Utils`